dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -187,8 +187,19 @@ interface RequestApi {
     ): ResponseEntity<StoredDataRequest>
 
     /** A method for searching data requests based on filters.
+     * @param dataType If set, only the requests with are data type in dataType are returned
+     * @param userId If set, only the requests from this user are returned
+     * @param emailAddress If set, only the requests from users which email address partially matches emailAddress are
+     *  returned
+     * @param requestStatus If set, only the requests with a request status in requestStatus are returned
+     * @param accessStatus If set, only the requests with an access status in accessStatus are returned
+     * @param reportingPeriod If set, only the requests with this reportingPeriod are returned
```

Comment:
If there is a use case for the request and access status to be Sets, does this not apply to reporting period, too?

Reply:
This is not needed in the frontend. Therefore it is not implemented.

Reply:
ok